java/flink/src/main/java/com/logicalclocks/hsfs/flink/HopsworksConnection.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2022. Hopsworks AB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ *  See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+package com.logicalclocks.hsfs.flink;
+
+import com.logicalclocks.hsfs.FeatureStoreException;
+import com.logicalclocks.hsfs.HopsworksConnectionBase;
+import com.logicalclocks.hsfs.SecretStore;
+import com.logicalclocks.hsfs.metadata.HopsworksClient;
+
+import lombok.Builder;
+
+import software.amazon.awssdk.regions.Region;
+
+import java.io.IOException;
+
+public class HopsworksConnection extends HopsworksConnectionBase {
+
+  @Builder
```

Comment:
For whatever reason in my example I can't use the builder pattern. I have to create the full HopsworksConnection manually. 

Looks like the builder is not public so I can't use it from outside.

Reply:
hmm, not sure what is the problem. could not reproduce. I could call this outside